hyperde3.mod

```diff
@@ -102,9 +102,6 @@ INITIAL {
       hys = hysinf
 	hyhtf = hyhtfinf
 	hyhts = hyhtsinf
-	VERBATIM
-	return 0;
-	ENDVERBATIM
```

Comment:
the generated code for this mod file is like this:

<img width="266" alt="image" src="https://user-images.githubusercontent.com/666852/203834735-74a0a646-2846-4561-b4bf-d61ee14bd669.png">


By quick look, I can't immediately say why the user wanted to return without `restoring the value of t`. Maybe @nrnhines might have some idea?

Otherwise, I would replace `return 0` with just `return` (to keep the same logic).

Reply:
Ok I will try `return`